.github/workflows/sycl_containers.yaml

```diff
@@ -10,11 +10,15 @@ on:
     paths:
       - 'devops/containers/**'
       - 'devops/dependencies.json'
+      - 'devops/scripts/install_drivers.sh'
+      - 'devops/scripts/install_build_tools.sh'
```

Comment:
Idea for future refactoring. It make sense either move scripts for configuring containers to `devops/containers/**` or come up with some naming scheme, so we can use wildcards to reduce amount of changes in this file.